utils/fork-tree/src/lib.rs

```diff
@@ -274,7 +271,8 @@ impl<H, N, V> ForkTree<H, N, V> where
 		number: &N,
 		is_descendent_of: &F,
 		predicate: &P,
-	) -> Result<Option<&Node<H, N, V>>, Error<E>> where
+	) -> Result<Option<&Node<H, N, V>>, Error<E>>
+	where
```

Comment:
I don't think there's a rustfmt setting that puts the `where` on the same line as the result type unfortunately.